tests/benchmarks/test_bench_e2e_pipelines.py

```diff
@@ -127,7 +127,7 @@ def ae_pipeline(config: Config, input_glob, repeat, train_data_glob, output_file
     pipeline.run()
 
 
-@pytest.mark.slow
+@pytest.mark.benchmark
```

Comment:
FYI, you can mark an entire module with:

```python
pytestmark = pytest.mark.benchmark
```
Just a heads up for the future. Docs are here: https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules